gprofiler/main.py

```diff
@@ -977,9 +977,12 @@ def main() -> None:
     if args.databricks_job_name_as_service_name:
         # "databricks" will be the default name in case of failure with --databricks-job-name-as-service-name flag
         args.service_name = "databricks"
-        databricks_client = DatabricksClient(logger)
-        if databricks_client.job_name is not None:
-            args.service_name = f"databricks-{databricks_client.job_name}"
+        dbx_web_ui_wrapper = DBXWebUIEnvWrapper(logger)
+        if (
+            dbx_web_ui_wrapper is not None
+            and (service_suffix := get_name_from_metadata(dbx_web_ui_wrapper.all_props_dict)) is not None
+        ):
```

Comment:
This line is too long and unreadable, please split it

Reply:
7342aaa